### DISCUSSION TITLE
Plot misorientation axis distribution

### DISCUSSION THREAD

**Discussion by Sritapaswi** [mtex-toolbox/mtex#728] [Ask Anything]
Hi all,

I am working with far-field high-energy diffraction microscopy, a.k.a 3D-XRD data. I am interested in plotting the misorientation axis distribution on an inverse pole figure as multiples of random distribution (MRD) for specific misorientation angles. I already calculated misorientation angles and axes for my data. I have the information in the form of quaternions, which I can convert to Euler angles if needed. Please guide me with the procedure for plotting the axis distribution. 

I saw that there is a function that does this - "plotaxisdistribution." However, I am not sure about the format of the input to achieve this. Please let me know.

Thanks,
Sri Tapaswi Nori

**Comment by kilir** [comment]
Hi Sri,
if you have your data in a text file with Euler angles in columns, you can use `orientation.load()`. See here: https://mtex-toolbox.github.io/orientation.load.html
Setting ss as crystalSymmetry should tell mtex that the data are actually misorientations.

If you like to have distributions of misorientation axes for different angles you should have a look at plotSection
https://mtex-toolbox.github.io/orientation.plotSection.html 

Hope that helps.

Cheers,
Rüdiger

**Reply by Sritapaswi** [reply]
Hello Rüdiger,

Thanks for your reply. It looks like "plotSection" does not work in my case. I am looking to represent the distribution of rotation axes as multiples of random distribution (MRD) on a stereographic projection for a given set of misorientations. Please refer to Figure 5 in the following journal article.
https://link.springer.com/content/pdf/10.1007/s11661-016-3831-x.pdf

Thanks,
Sri Tapaswi Nori

**Reply by kilir** [reply]
Hi Sri,
if you want densities, you can contour your plot or use `calcDensity` to a) compute an MDF for your misorientations to plot your sections or b) select the misorientations with the angles of interest and compute the density of the corresponding axes. The result will most likely be different.

More interestingly, how would you define your random distribution? Something like a uniform axis distribution of axes or according to a random axis sample form a uniform MDF? It's not clear from your reference.

Cheers,
Rüdiger

**Reply by Sritapaswi** [reply]
Hi Rüdiger,

Thanks for the reply. As you mentioned, it will be interesting to understand how random distribution is defined. More interestingly, I found similar plots done in one of the older mtex documents as indicated in the link below. Please refer to the section of **The axis distribution** that uses the function "plotaxisdistribution" albeit the plots encompass all the misorientation angles. Can you please explain how random distribution is explained in these plots so that I can apply a similar logic for plotting the axis distribution for specific misorientation angles such as 60 degrees and 38.9 degrees?. If you think the link I shared can solve my question, can you please let me know, what is the data structure of the input for "plotaxisdistribution" so that I can put my data in the required format?

https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html#22

Thanks,
Sri Tapaswi Nori

**Reply by kilir** [reply]
Hi Sri,

have a look at the help of plotAxisDistribution (` help plotAxisDistribution`).

Cheers,
Rüdiger
